x-pack/plugins/security_solution/public/management/pages/policy/view/event_filters/list/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export { PolicyEventFiltersList } from './policy_event_filters_list';
```

Comment:
nitpick: I am not a fan of this pattern. Having files that simply re-export files from the same level. It makes it harder to navigate the code because things like "go to definition" will take you extra steps and creates additional files and modules without requirement.

Reply:
As the file names are a bit long this will avoid us to have very long import paths, for example:
Instead of `import whatever from 'public/management/pages/policy/view/event_filters/list/policy_event_filters_list.tsx'` it will be `import whatever from 'public/management/pages/policy/view/event_filters/list'`

Reply:
IMO: a long import path is better than a web of re-exports that are harder to follow.